poly2tri/common/shapes.cc

```diff
@@ -243,15 +243,28 @@ Point* Triangle::PointCCW(const Point& point)
   return nullptr;
 }
 
+// The neighbor across to given point
+Triangle* Triangle::NeighborAcross(const Point& point)
+{
+  if (&point == points_[0]) {
+    return neighbors_[0];
+  } else if (&point == points_[1]) {
+    return neighbors_[1];
+  } else {
+    return neighbors_[2];
+  }
+}
+
 // The neighbor clockwise to given point
 Triangle* Triangle::NeighborCW(const Point& point)
 {
   if (&point == points_[0]) {
     return neighbors_[1];
   } else if (&point == points_[1]) {
     return neighbors_[2];
+  } else {
```

Comment:
I don't like this change, see https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html
Can you keep the return without else?

Reply:
Sure, no problem. Thanks for the link!